### ISSUE TITLE
3D Benchy

### ISSUE THREAD

**Issue by Vlek** [Vlek/3D_Printing#36]
https://www.thingiverse.com/thing:763622

3DBenchy.stl
10 mb | Updated 04-09-2015 | 3286813 Downloads

**Comment by Vlek** [comment]
Second attempt for #22.

I believe I have the z-offset set correctly finally.

I have also increased the print speed to 120% just to see this thing rock and roll.

**Comment by Vlek** [comment]
This printed and the quality is identical for the most part to the first.

The first layer is marginally better. I am getting closer to dialing it in.

I think the problem is that I did not zero out the z axis first before doing my z-offset, so the number that I came to was off.

The actual steps are like:
- Ensure that the hotend is clear of any filament blobs that will skew the results
- Set z-axis offset to 1 or 2 so that moving the z to 0 doesn't hit.
- Move z-axis to 0.
- Make medium-sized adjustments down until the hotend gets close to the bed
- Make small adjustments down while moving a piece of paper to see if it catches
- Take note of the z-offset once it slightly catches the paper
